MyJeopardy.js

```diff
@@ -35,7 +35,7 @@ async function getClues(catId) {
  *   each with a question for each category in a <td>
  *   (initally, just show a "?" where the question/answer would go.)
  */
-async function jeopardyGB () {
+async function populateGameBoard () {
```

Comment:
`jeopardyGB` isn't a good function name. Functions should be name according to the actions they perform.

Reply:
Understood.